apps/etterlatte-behandling/src/main/kotlin/behandling/domain/Behandling.kt

```diff
@@ -174,7 +175,31 @@ sealed class Behandling {
     ) : Exception(message)
 }
 
-internal fun Behandling.toDetaljertBehandling(persongalleri: Persongalleri): DetaljertBehandling {
+internal fun Behandling.toStatistikkBehandling(persongalleri: Persongalleri): StatistikkBehandling {
+    return StatistikkBehandling(
+        id = id,
+        sak = sak,
+        sistEndret = sistEndret,
+        behandlingOpprettet = behandlingOpprettet,
+        soeknadMottattDato = mottattDato(),
+        innsender = persongalleri.innsender,
+        soeker = persongalleri.soeker,
+        gjenlevende = persongalleri.gjenlevende,
+        avdoed = persongalleri.avdoed,
+        soesken = persongalleri.soesken,
+        status = status,
+        behandlingType = type,
+        virkningstidspunkt = virkningstidspunkt,
+        boddEllerArbeidetUtlandet = boddEllerArbeidetUtlandet,
+        revurderingsaarsak = revurderingsaarsak(),
+        prosesstype = prosesstype,
+        revurderingInfo = revurderingInfo()?.revurderingInfo,
+        enhet = sak.enhet,
+        kilde = kilde,
+    )
+}
+
+internal fun Behandling.toDetaljertBehandlingFromPersongalleri(persongalleri: Persongalleri): DetaljertBehandling {
```

Comment:
Å kalle den `...FromPersongalleri` synes jeg blir litt misvisende. For du lager ikke behandlingen _fra persongalleriet_. Du konverterer en `Behandling` til `DetaljertBehandling` og legger til persongalleri.

Reply:
Hva med `toDetaljertBehandlingWithPersongalleri`?

Reply:
For meg var det implisitt sånn som det var. Mye kortere funksjonsnavn også. 

`it.toDetaljertBehandling(persongalleri)` forteller meg at objektet konverteres og at data fra persongalleriet brukes i konverteringen. Eneste som er uheldig her er bruken av `it` siden man da må ha i bakhodet hva _it_ er. 

Hvis man først skal skrive om noe her burde det heller legges på detaljertbehandling-objektet som companion. 
Da kan man skrive `DetaljertBehandling.fra(behandling, persongalleri)`.